examples/package.json

```
@@ -8,21 +8,21 @@
     "formatter": "prettier --check .",
     "test": "npm run linter && npm run typecheck && npm run formatter",
     "clean-examples": "rm -rf ./public ../.parcel-cache",
-    "start:example": "npm run clean-examples && cross-env $(cat .env) PARCEL_AUTOINSTALL=false parcel serve $EXAMPLE_ENTRY --dist-dir public --port 1234 --no-cache",
-    "start:map": "EXAMPLE_ENTRY=./basic-google-map/index.html npm run start:example",
-    "start:map-with-markers": "EXAMPLE_ENTRY=./google-map-with-markers/index.html npm run start:example",
-    "start:multiple-maps": "EXAMPLE_ENTRY=./multiple-google-maps/index.html npm run start:example",
-    "start:geocoding-service": "EXAMPLE_ENTRY=./geocoding-service/index.html npm run start:example",
-    "start:places-service": "EXAMPLE_ENTRY=./places-service/index.html npm run start:example",
-    "start:places-service-with-element": "EXAMPLE_ENTRY=./places-service-with-element/index.html npm run start:example",
-    "start:places-autocomplete-widget": "EXAMPLE_ENTRY=./places-autocomplete-widget/index.html npm run start:example",
-    "start:directions-service": "EXAMPLE_ENTRY=./directions-service/index.html npm run start:example",
-    "start:distance-matrix-service": "EXAMPLE_ENTRY=./distance-matrix-service/index.html npm run start:example",
-    "start:elevation-service": "EXAMPLE_ENTRY=./elevation-service/index.html npm run start:example",
-    "start:max-zoom-service": "EXAMPLE_ENTRY=./max-zoom-service/index.html npm run start:example",
-    "start:places-autocomplete-service": "EXAMPLE_ENTRY=./places-autocomplete-service/index.html npm run start:example",
-    "start:street-view-panorama-map": "EXAMPLE_ENTRY=./street-view-panorama-map/index.html npm run start:example",
-    "start:street-view-panorama-element": "EXAMPLE_ENTRY=./street-view-panorama-with-element/index.html npm run start:example",
+    "start:example": "npm run clean-examples && cross-env $(cat .env) cross-env PARCEL_AUTOINSTALL=false parcel serve $EXAMPLE_ENTRY --dist-dir public --port 1234 --no-cache",
```

```suggestion
    "start:example": "npm run clean-examples && cross-env $(cat .env) PARCEL_AUTOINSTALL=false parcel serve $EXAMPLE_ENTRY --dist-dir public --port 1234 --no-cache",
```

Shouldn't be needed twice (see https://github.com/kentcdodds/cross-env/issues/15)